retroshare.pri

```diff
@@ -434,6 +434,8 @@ no_rs_deprecatedwarning {
 
 no_rs_cppwarning {
     QMAKE_CXXFLAGS += -Wno-cpp
+    QMAKE_CXXFLAGS += -Wno-inconsistent-missing-override
```

Comment:
I think this warning is useful and we shouldn't suppress it, I have started adding override keyword when appropriate on the code I touch

Reply:
This is removed only with no_rs_cppwarning flag.
So it will be removed when less warnings exists. For now it pollute compiler log.